packages/react/src/components/Forms/DatePicker/index.scss

```diff
@@ -0,0 +1,255 @@
+      border-radius: var(--c--components--forms-datepicker--border-radius);
+
+      &__toggle {
+        color: var(--c--theme--colors--danger-600);
+      }
+
+      label {
+        color: var(--c--theme--colors--danger-600);
+      }
+    }
+
+    &:not(.c__date-picker--disabled) {
+      &:hover {
+        .c__date-picker__wrapper {
+          border-color: var(--c--theme--colors--danger-200);
+          &__toggle {
+            color: var(--c--theme--colors--danger-200);
+          }
+        }
+      }
+    }
+  }
+
+  &--picked {
+    .c__date-picker__wrapper {
+      border-radius: var(--c--components--forms-datepicker--border-radius--focus);
+      border-color: var(--c--components--forms-datepicker--border-color--focus);
+
+      &__toggle {
+        color: var(--c--components--forms-datepicker--border-color--focus);
+      }
+    }}
+
+  &:not(&--picked):not(&--invalid):not(&--disabled) {
+    &:hover {
+      .c__date-picker__wrapper {
+        border-radius: var(--c--components--forms-datepicker--border-radius--hover);
+        border-color: var(--c--components--forms-datepicker--border-color--hover);
+      }
+    }
+  }
+
+}
+
+
+.c__calendar {
+  display: block;
+  transform: translate(2px, 0);
+
+  &__wrapper {
+    display: block;
+    padding: 0.75em;
+
+    &--opened {
+      box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.3);
+      background-color: var(--c--components--forms-datepicker--menu-background-color);
+    }
+
+    &__header {
+      display: flex;
+      justify-content: space-between;
+
+      &__actions {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        overflow: hidden;
+        white-space: nowrap;
+        min-width: 180px;
+      }
+    }
+
+    &__grid {
+      width: 100%;
+      padding-top: 0.5em;
+
+      &__header-row {
+        line-height: 4em;
+        font-size: var(--c--components--button--small-font-size);
+
+        th {
+          font-weight: var(--c--components--button--font-weight);
+          color: var(--c--theme--colors--greyscale-700);
+        }
+      }
+
+      &__week-row {
+        &__button {
+          margin: 0.3em auto;
+          &--selected {
+            border-radius: 100%;
+          }
+        }
+        .c__button--small {
+          height: 27.63px;
+          width: 27.63px;
+          padding: 0;
+          align-items: center;
+          justify-content: center;
+        }
+      }
+    }
+  }
+
+  &__menu {
+    position: absolute;
+    top: 0;
+    left: -1px;
+    overflow: auto;
+    width: 100%;
+    max-height: 25rem;
+    box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.3);
+    background-color: var(--c--components--forms-datepicker--menu-background-color);
+    transform: translate(2px, 0);
+    display: none;
+    z-index: 2;
+
+    &--opened {
+      display: block;
+    }
+
+    ul {
+      list-style-type: none;
+      padding: 0;
+      margin: 0;
+      padding-top: 3px;
```

Comment:
We try to use rem as much as possible.

Reply:
this style actually is duplicated from Select stylesheet [l.108](https://github.com/openfun/cunningham/blob/9065d6c87bceaff910583dd070b84dc335827ac4/packages/react/src/components/Forms/Select/index.scss#L108)

Reply:
Hmm after thinking about it, I think that for such magic small values px is maybe the best suited one for DX. What do you think @jbpenrath ?

Reply:
I like rem to scale space with font size base.
But in order to keep a good dx, we could have a `rem-calc` util which allow us to use pixel value.